tycho-baseline-plugin/src/main/java/org/eclipse/tycho/baseline/BaselineContext.java

```diff
@@ -26,6 +26,8 @@ public interface BaselineContext {
 	void reportBaselineProblem(String message) throws MojoFailureException;
 
 	List<String> getIgnores();
+	
+	List<String> getTextFileExtensions();
```

Comment:
is this really something we like to configure?

Reply:
If you know a reliable way to detect if a file is a text-file then it does not need to be configurable. But since I don't know any my solution currently is to provide a good default set of text-file-extensions (it can probably be enhanced) and let the user adjust it if that does not match that users reality.